js/common.d.ts

```diff
@@ -343,6 +343,10 @@ export type GlobalConfig = {
    * @public
    */
   oDataFilterToLower?: boolean;
+  /**
+   * @deprecated
```

Comment:
Please add the text

Attention! This field is not documented and should only be specified in a limited number of use cases. For more information, please submit a ticket to our {@link https://supportcenter.devexpress.com/ticket/create Support Center}.

Reply:
Agree